lib/testing/src/lib/content-services/dialog/content-node-selector-dialog.page.ts

```diff
@@ -95,6 +100,14 @@ export class ContentNodeSelectorDialogPage {
         await this.contentList.dataTablePage().clickRowByContent(name);
     }
 
+    async doubleClickContentNodeSelectorResult(name): Promise<void> {
+        await this.contentList.dataTablePage().doubleClickRowByContent(name);
+    }
+
+    async doubleClickContentNodeFolder(name): Promise<void> {
+        await this.doubleClickContentNodeSelectorResult(name);
```

Comment:
doubleClickContentNodeFolder just call doubleClickContentNodeSelectorResult is it really needed ?